### ISSUE TITLE
Prevents ad blocking  TwitchNoSub 0.8

### ISSUE THREAD

**Issue by 0sryder** [besuper/TwitchNoSub#138]
Ads are no longer blocked by ublock origin with the advanced setting : 

userResourcesLocation : 
https://raw.githubusercontent.com/pixeltris/TwitchAdSolutions/master/video-swap-new/video-swap-new-ublock-origin.js

And the customized filter: twitch.tv##+js(twitch-videoad)

Edge 131.0.2903.70

The only (temporary) solution is to delete all cookies from Twitch.tv and use the old version of TwitchNoSub 0.7.1.

**Comment by fREAST** [comment]
Same issue with the vaft userscript from https://github.com/pixeltris/TwitchAdSolutions/blob/master/vaft/vaft.user.js

Using TwitchNoSub 0.8, it fails to load with `Twitch Worker is already hooked. Skipping (vaft)`, with 0.7.1 it works after clearing cookies/site data, however it stops working after a couple of days and I have to clear site data etc again otherwise streams just show a black screen.

**Comment by pixeltris** [comment]
Duplicate of #121

It's due to the conflicting Worker hook. Something that should probably be fixed in the vaft/video-swap-new scripts.

**Comment by mehhxyeah** [comment]
> Same issue with the vaft userscript from https://github.com/pixeltris/TwitchAdSolutions/blob/master/vaft/vaft.user.js
> 
> Using TwitchNoSub 0.8, it fails to load with `Twitch Worker is already hooked. Skipping (vaft)`, with 0.7.1 it works after clearing cookies/site data, however it stops working after a couple of days and I have to clear site data etc again otherwise streams just show a black screen.

For me the clearing of cookies / site data doesn't work anymore even on v0.71 :/, jsut cant block ads anymore if you wna use this extension at least for now

**Comment by BlueWorld1** [comment]
Same issue here, I just removed the code that detect if there is multiple adblocker enabled, and I' m sure that I only have one.

Code to remove:
` if (isWorkerDoubleHooked(newWorker, 'twitch')) {
                    console.log('Multiple twitch adblockers installed. Skipping Worker hook (vaft)');
                    isTwitchWorker = false;
                }`
Or just make the function isWorkerDoubleHooked always return true.

Obviously, this is not the fix, we need to wait for the author to really fix the DOUBLE adblocker detection/worker injection.
Obviously, if this code exist, it's probably because you can have bad behaviour or side effect by enabling 2 scripts that inject some behaviour into the same player. Make sure you only have one script enabled. 
I've found that we hook twice the worker (function hookWindowWorker is executed twice) but for now I did not experience any bug, it's probably not efficient, but no significant impact for now.

Since this change, everything works are before with 0.8 and chrome.

**Comment by pixeltris** [comment]
Fixed in https://github.com/pixeltris/TwitchAdSolutions/commit/43a03975047dc1ecc2478f08c226c3c7f34d043f
